src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1593,7 +1607,7 @@ internal void DisconnectOutput(int portData, int inPortData, NodeModel nodeModel
 
         private void ClearTooltipText()
         {
-            ToolTipText = "";
+            ToolTipText = string.Empty;
```

Comment:
`ToolTipText `has been deprecated in PR #12712, we are now only using `Infos` as the single source of truth (assuming we merge #12712 before this one of course).

Reply:
Removed this statement.